package.json

```diff
@@ -91,7 +91,8 @@
     "i18next-scanner": "2.11.0",
     "ngrok": "3.2.7",
     "prettier": "^1.18.2",
-    "release-it": "^13.6.2"
+    "release-it": "^13.6.2",
+    "twilio-run": "^2.7.0"
```

Comment:
it would be awesome to have an npm script in this root package.json to run twilio locally like the other apps (so `local:twilio`).

it wouldn't have to run the express index.js, since twilio-run supports ngrok and the like. we'd just have to point twilio-run to the src/twilio directory

Reply:
cool good idea!
(i made it default to running ngrok, because that's really the only way to test via a dev number ... does that make sense?)